### PR TITLE
Reformatted the comments into a table

### PR DESCRIPTION
**Pull request by lattai**
Visit at https://leahattai-step-2020.appspot.com/comments.html

### CODE REVIEW

File: portfolio/src/main/webapp/comments-script.js [hunk 1]
```diff
@@ -16,28 +16,37 @@
  * Fetches the comments input and builds the UI into a list.
  */
 
+var row;
 function getComments() {
-    const responsePromise = fetch('/comments');
-    responsePromise.then(handleResponse);
+    const RESPONSE_PROMISE = fetch('/comments');
```
Comment (WillHipschman) [inline portfolio/src/main/webapp/comments-script.js:21]: I don't think you need to use UPPER_SNAKE_CASE for these variables. They are constant (can't be re-assigned to) but the rule for UPPER_SNAKE_CASE is that the type should be deeply immutable and statically (class) scoped.

File: portfolio/src/main/webapp/comments-script.js [hunk 2]
```diff
@@ -16,28 +16,37 @@
...
+    const TABLE_ELEMENT = document.getElementById('comments-table');
+    TABLE_ELEMENT.innerHTML = '<tr><th>Name</th><th>Message</th></tr>';
     for (var i = 0; i < comments.length; i ++) {
-        commentsElement.appendChild(
-        createListElement(comments[i].name + " - " + comments[i].message));
+        row = createRowElement();
+        row.appendChild(createDataElement(comments[i].name));
+        row.appendChild(createDataElement(comments[i].message));
+        TABLE_ELEMENT.appendChild(row);
     }
```
Comment (DirtGrubDylan) [inline portfolio/src/main/webapp/comments-script.js:38]: You should consider a more functional approach here: http://cryto.net/~joepie91/blog/2015/05/04/functional-programming-in-javascript-map-filter-reduce/

File: portfolio/src/main/webapp/comments-script.js (hunk 1 above)
Comment (WillHipschman) [reply portfolio/src/main/webapp/comments-script.js:21]: See https://google.github.io/styleguide/jsguide.html#naming-constant-names

File: portfolio/src/main/webapp/comments-script.js [hunk 3]
```diff
@@ -16,28 +16,37 @@
...
-    JsonPromise.then(addCommentToDom);
+    const JSON_PROMISE = response.json();
+    JSON_PROMISE.then(addCommentToDom);
 }
 
 function addCommentToDom(comments) {
-    const commentsElement = document.getElementById('comments-container');
-    commentsElement.innerHTML = " ";
+    const TABLE_ELEMENT = document.getElementById('comments-table');
+    TABLE_ELEMENT.innerHTML = '<tr><th>Name</th><th>Message</th></tr>';
```
Comment (WillHipschman) [inline portfolio/src/main/webapp/comments-script.js:32]: Here too. You declare it as a constant and then immediately re-assign a sub-object. It is not deeply immutable.

**Pull request merged**